test.js

```diff
@@ -22,7 +22,9 @@ describe('millisecond', function () {
   });
 
   it('should return 0 if invalid', function () {
-    assume(ms('Hello mom')).to.equal(0);
+    var s = ms('[/\]')
+    console.log(s)
```

Comment:
I would suggest removing this debug code, keeping the original assume and add your commented out assume statement.